### PR TITLE
refactor: create an instance

### PR DESCRIPTION
**Pull request by Lightning00Blade**
Comments on places where changes were made.

### CODE REVIEW

File: src/bidiServer/WebSocketServer.ts [hunk 1]
```diff
@@ -55,333 +55,337 @@ type SessionOptions = {
...
+    this.#channel = channel;
+    this.#headless = headless;
+    this.#verbose = verbose;
+
+    this.#server = http.createServer(this.#onRequest.bind(this));
+    this.#wsServer = new websocket.server({
+      httpServer: this.#server,
+      autoAcceptConnections: false,
+    });
+    this.#wsServer.on('request', this.#onWsRequest.bind(this));
```
Comment (Lightning00Blade) [inline src/bidiServer/WebSocketServer.ts:83]: Extract into separate functions.

File: src/bidiServer/WebSocketServer.ts [hunk 2]
```diff
@@ -55,333 +55,337 @@ type SessionOptions = {
...
+    }
 
-    const wsServer: websocket.server = new websocket.server({
-      httpServer: server,
-      autoAcceptConnections: false,
-    });
+    // https://w3c.github.io/webdriver-bidi/#transport, step 2.
+    if (request.url === '/session') {
+      const bodyArray: Uint8Array[] = [];
+      const body = await new Promise<Buffer>((resolve, reject) => {
```
Comment (Lightning00Blade) [inline src/bidiServer/WebSocketServer.ts:106]: Created a promise to reduce nesting.

File: wpt
Comment (OrKoN) [inline wpt:1]: exclude wpt?

Comment (Lightning00Blade) [reply wpt:1]: Strange my IDE usually does this automatically.  Removing.

**Review by sadym-chromium — APPROVED** [review]
Even though the reviewing of the whole changes are quite challenging, I assume it is no-op change. As long as the NodeJS Mapper runner is used only in CI, I believe it's safe to proceed.

**Pull request merged**